### PR TITLE
Add evaluation endpoint

### PR DESCRIPTION
**Pull request by JonasPetry**
This PR adds the evaluation endpoint to the tracking service

### CODE REVIEW

File: tracking/src/main.py [hunk 1]
```diff
@@ -30,3 +37,215 @@ async def testPost(request: Request, response: Response):
...
+
+    return timedelta.total_seconds()
+
+
+def classify_comment(s1: str, s2: str):
+    if s1 == s2:
+        return FeedbackType.Automatic
+    elif levenshtein(s1, s2) > 0.9:
+        return FeedbackType.Typo
+    elif lcs(s1, s2) > 0.95 and jaro_winkler(s1, s2) > 0.6:
```
Comment (bigultekin) [inline tracking/src/main.py:161]: How did you find these numbers?

File: tracking/src/main.py [hunk 2]
```diff
@@ -30,3 +37,215 @@ async def testPost(request: Request, response: Response):
...
+                    assessed_participations.append(participation_id)
+        else:
+            if df_reference.head(1)['feedbacks.type'].values[0] == 'MANUAL':
+                log_count += 1
+
+    percentage_provided = round(len(score_first_feedbacks) / count * 100, 2)
+
+    # print(log_count)
+
+    kappa_val = cohens_kappa(score_first_feedbacks, score_last_feedbacks)
```
Comment (bigultekin) [inline tracking/src/main.py:225]: Nice! What's the kappa usually?

**Review by bigultekin — APPROVED** [review]
I didn't run the code but metrics look good. I liked it. Nice job 👍

File: tracking/src/main.py (hunk 1 above)
Comment (JonasPetry) [reply tracking/src/main.py:161]: They are somewhat arbitraty. The important point is that they stay the same over time

File: tracking/src/main.py (hunk 2 above)
Comment (JonasPetry) [reply tracking/src/main.py:225]: It is considered good at 0.7 and has a range of -1 to 1

Comment (bigultekin) [reply tracking/src/main.py:225]: I know, I meant the value you found for the text exercises so far

**Pull request merged**